cmd/release/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+# flag for Binary name
+if [ "$1" != "" ]; then
+  cmd=$1
+  else
+    echo "Please specify binary name"
+    exit 1
+fi
+
+# flag for OS, CGO_ENABLED=1 means cross compile is not available
+if [ "$2" != "" ]; then
+  os=$2
+  else
+    echo "Please specify OS"
+    exit 1
+fi
+
+# flag for arch, CGO_ENABLED=1 means cross compile is not available
+if [ "$3" != "" ]; then
+  arch=$3
+  else
+    echo "Please specify arch"
+    exit 1
+fi
+
+GIT_COMMIT=$(git rev-list -1 HEAD)
+GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+GIT_TAG=$(git describe --tags '--match=v*' --dirty)
+
+export CGO_CFLAGS="$(go env CGO_CFLAGS 2>/dev/null) -DMDBX_FORCE_ASSERTIONS=0 -O -D__BLST_PORTABLE__ -Wno-error=strict-prototypes"
+
+BUILD_TAGS=nosqlite,noboltdb,disable_libutp
+PACKAGE=github.com/ledgerwatch/erigon
+
+NAME="${cmd}"_"${os}"_"${arch}"
+OUTDIR=$(pwd)/build/bin/"${GIT_COMMIT}"/"${NAME}"
+CURRENTDIR=$(pwd)
+PASSFILE=$(pwd)/passphrase.txt
+
+mkdir -p "${OUTDIR}"
+
+cd "$CURRENTDIR"/cmd/"${cmd}" || exit 1
+echo "building ${cmd} for ${os} ${arch}"
+GOOS="${os}" GOARCH="${arch}" CGO_CFLAGS="$(go env CGO_CFLAGS 2>/dev/null) -DMDBX_FORCE_ASSERTIONS=0 -O -D__BLST_PORTABLE__ -Wno-error=strict-prototypes" /usr/local/go/bin/go build -o "${OUTDIR}"/"${NAME}" $GO_FLAGS -trimpath -tags ${BUILD_TAGS} -buildvcs=false -ldflags "-X ${PACKAGE}/params.GitCommit=${GIT_COMMIT} -X ${PACKAGE}/params.GitBranch=${GIT_BRANCH} -X ${PACKAGE}/params.GitTag=${GIT_TAG}"
```

Comment:
@AskAlexSharov please could you confirm these are suitable flags for production release builds? :)

Reply:
yes. but we have this list in Makefile already. Maybe better get it from there. Possible without copy-paste?
There is `BUILD_TAGS`, `GO_FLAGS`, `CGO_CFLAGS`, etc...

Reply:
FYI: we must use same C flags when `make db-tools`

Reply:
Thanks @AskAlexSharov the reason I moved from there was because the Makefile comprises docker, dev builds, etc. and wanted to make sure this was purely release builds only to reduce noise.

Do we need to provide binaries for db-tools as well?